### ISSUE TITLE
Feature: Renderer System for Three.js

### ISSUE THREAD

**Issue by DavidPeicho** [DavidPeicho/ecstra#3]
* [ ] Renderer system should go in a Rendering group, i.e., post-update
* [ ] Renderer should be able to render to one or more views.
* [ ] Views can be a subregions of framebuffer, or a totally different context

**Comment by zlpersonal** [comment]
Will you be make this as sort of an extension? I use ECSY + Cocos Creator for rendering. It'll be good if you could separate the render system into another project like https://github.com/ecsyjs/ecsy-three.

**Comment by DavidPeicho** [comment]
Yes, definitely will not be part of the core like that. I am just trying to write my ideas and sort them properly 😄
